content/en/docs/how-tos/adding-a-cluster-profile.md

```diff
@@ -11,18 +11,18 @@ about platform enablement process, please see the [OpenShift Infrastructure Prov
 
 {{< /alert >}}
 
-## What a Cluster Profile?
+## What Is a Cluster Profile?
```

Comment:
nit:
```suggestion
## What is a Cluster Profile?
```

Reply:
Hmm, is that right? (genuine question, I don't know)

We use title case in these pages, and it looks like it is normally capitalized:

- https://en.wikipedia.org/w/index.php?search=what+is
- https://duckduckgo.com/?q=%22what+is%22&t=ffab&iar=news&ia=news

Reply:
huh, who knew? it doesn't look right capitalized to me, but...https://cmosshoptalk.com/2021/08/24/is-is-always-capitalized-in-titles/#:~:text=What's%20the%20Verdict%3F,a%20Chicago%2Dstyle%20source%20citation.

Reply:
we should've just went with "What's" ;)

Reply:
XD